docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -247,6 +227,72 @@ To sum this up in pseudo algorithms:
   total_health = min(100, Σ(health(P_0)...health(P_X))
   load to P_X = 100 - Σ(percent_load(P_0)..percent_load(P_X-1))
 
+.. _arch_overview_load_balancing_panic_threshold:
```

Comment:
This change is definitely worthy of a release note. Can you add one?

Reply:
Sure, where is file with release notes?

Reply:
https://github.com/envoyproxy/envoy/blob/master/docs/root/intro/version_history.rst